config/settings.yml

```diff
@@ -855,6 +855,7 @@ flipper:
     - brian.seek@civicactions.com # Brian Seek (Civic Actions), FE Engineer Health Care Experience/Check-In
     - clark@governmentcio.com # Designer on GCIO
     - cameron@oddball.io
+    - casey.williams@adhocteam.us # Backend Engineer, Benefits & Appeals
```

Comment:
I guess you would need a valid IDME account tied to this email in addition to this setting ?